integration/basic_test.go

```diff
@@ -524,7 +524,7 @@ func TestDatabaseName(t *testing.T) {
 		ctx, collection := setup.Setup(t)
 
 		err := collection.Database().Client().Database("").CreateCollection(ctx, collection.Name())
-		expectedErr := driver.InvalidOperationError(driver.InvalidOperationError{MissingField: "Database"})
+		expectedErr := errors.New("database name cannot be empty")
```

Comment:
That error is returned by the driver, not FerretDB or MongoDB. We should just delete that test.

Reply:
That's right

Reply:
We could, in theory, write a test that uses `RunCommand` to do `insert`… But let's not do that now.